packages/fiori/src/Wizard.js

```diff
@@ -803,9 +834,7 @@ class Wizard extends UI5Element {
 				activeSeparator: (idx < lastEnabledStepIndex) && !step.disabled,
 				branchingSeparator: step.branching,
 				pos,
-				size: stepsCount,
-				roleDescription,
-				ariaLabel: getEffectiveAriaLabelText(step),
```

Comment:
Previously the app developers could write aria-label on the abstract ui5-wizard-step

```html
<ui5-wizard>
    <ui5-wizard-step aria-label="Hello">
```

which is then rendered at the respective tab in the header navigation (within the Shadow DOM)

```html
<ui5-wizard-tab  aria-label="Hello"></ui5-wizard-tab>
```

Is it still possible after this change?